packages/create-neutrino-project/commands/init/templates/neutrino-preset-vue/src/index.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue';
+import App from './App.vue';
+
```

Comment:
Is there anything we can do here for HMR?

Reply:
`vue-loader` has HMR integrated within haha. https://vue-loader.vuejs.org/en/features/hot-reload.html

There's no need to add `if (module.hot)...`. The `<template>, <script>, <style>` in vue files does it out of the box when you have `vue-loader`.

Reply:
😍 😍 😍 😍 😍 😍 😍 😍